packages/orderbook/src/config/config.ts

```diff
@@ -17,7 +17,7 @@ export interface OrderbookModuleConfiguration {
   zoneContractAddress: string;
   apiEndpoint: string;
   chainName: string;
-  provider: providers.JsonRpcProvider | providers.Web3Provider;
```

Comment:
Same here - wouldn't it break backwards compatibility if someone was using a Web3Provider here?

Reply:
This module config type is not used when initiating the module. Only the base config and overrides can be provided